docs/DxcOnLinux.rst

```diff
@@ -0,0 +1,78 @@
+=======================================
+DirectXShaderCompiler on Linux & Mac OS
+=======================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+============
+
+Due to several `requests <https://github.com/Microsoft/DirectXShaderCompiler/issues/1236>`_
+from the community, we have started the effort to enable compilation and running
+of DirectXShaderCompiler on non-Windows platforms (Linux and Mac OS).
+
+
```

Comment:
Need a section to describe the prerequisites, like compiler requirements.

Reply:
Done